scripts/host-check

```diff
@@ -1199,6 +1304,8 @@ BASE_CHECKS = [
     Check("Core pattern", check_core_pattern, []),
     Check("ASLR", check_userspace_aslr, []),
     Check("Linux Security Modules", check_linux_security_modules, []),
+    Check("Socket kernel parameters", check_socket_parameters, []),
+    Check("UDP kernel parameters", check_udp_parameters, []),
```

Comment:
Perhaps put up with the inotify checks to group kernel config checks together? (maybe the inotify checks could be one check really...)